### PR TITLE
Use single URDF for both Pinocchio and ROS

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: include/panda_torque_mpc/common.h [hunk 1]
```diff
@@ -36,13 +38,13 @@ namespace panda_torque_mpc {
...
-    inline pinocchio::Model loadPandaPinocchio()
+    inline pinocchio::Model loadPandaPinocchio(const std::string& robot_description)
     {
         // Load panda model with pinocchio and example-robot-data
-        std::string urdf_path = ros::package::getPath("panda_torque_mpc") + "/urdf/robot.urdf";
         std::string srdf_path = ros::package::getPath("panda_torque_mpc") + "/srdf/demo.srdf";
         pinocchio::Model model_pin_full;
-        pinocchio::urdf::buildModel(urdf_path, model_pin_full);
+        const auto urdf_tree = urdf::parseURDF(robot_description);
+        pinocchio::urdf::buildModel(urdf_tree, model_pin_full);
```
Comment (MaximilienNaveau) [inline include/panda_torque_mpc/common.h:47]: For building the model in pinocchio you have a function called buildFromXML somewhere.
Look at [pinocchio/parser](https://github.com/stack-of-tasks/pinocchio/blob/v2.7.1/include/pinocchio/parsers/urdf.hpp#L135)

File: include/panda_torque_mpc/common.h [hunk 2]
```diff
@@ -11,6 +11,8 @@
 #include <geometry_msgs/Pose.h>
 #include <geometry_msgs/PoseStamped.h>
 
+#include <urdf_parser/urdf_parser.h>
```
Comment (MaximilienNaveau) [inline include/panda_torque_mpc/common.h:14]: wrong urdf usage.
Please use a different approach cited below to read the xml string directly.

**Review by MaximilienNaveau — CHANGES_REQUESTED** [review]

File: include/panda_torque_mpc/common.h (hunk 1 above)
Comment (Kotochleb) [reply include/panda_torque_mpc/common.h]: Fixed

File: include/panda_torque_mpc/common.h (hunk 2 above)
Comment (Kotochleb) [reply include/panda_torque_mpc/common.h]: Fixed

File: package.xml [hunk 3]
```diff
@@ -23,6 +23,7 @@
   <depend>eigen_conversions</depend>
   <depend>franka_hw</depend>
   <depend>franka_gripper</depend>
+  <depend>franka_ros</depend>
```
Comment (MaximilienNaveau) [inline package.xml:26]: why?

**Review by MaximilienNaveau — APPROVED** [review]
Just add a small question.
Please fell free to merge

File: package.xml (hunk 3 above)
Comment (Kotochleb) [reply package.xml:26]: Right. I forgot. This should be a separate PR. This is actually fix to a missing dependency

Comment (MaximilienNaveau) [reply package.xml:26]: leave it
let's merge!

**Comment by MaximilienNaveau** [comment]
@nim65s can you merge this?
For some reasons I do not have the merge button here...

**Pull request merged**